wiki/en/Running-a-Server.md

```diff
@@ -18,50 +18,39 @@ permalink: "/wiki/Running-a-Server"
 </details>
 
 
-
 ## Do I need to run a Server?
 
-The short answer is **no**. For various reasons, you should try to use Servers listed by the built in Directories, at least at first while getting your sound set up with the Jamulus Client. You may also be able to use a Server hosted by a third party. Doing so will save you the trouble of setting one up yourself.  
-
-Note that you can have a "private" session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
+**No**. You can use the Servers listed by the built in Directories and use Jamulus without running a server.
 
-If you decide you cannot use any of the Servers listed by the built in Directories, you may be able to use a Server (either Registered in a Custom Directory, or Unregistered - see [Server Types](#server-types)) hosted by a third party. Doing so will save you the trouble of setting one up yourself.
+**Note:** You can even have a "private", uninterrupted session with other people on a Server by simply soloing each other. You will then not be able to hear anyone else if they enter your Server (note that this does not stop them from hearing you, using the chat function, or seeing your profile information).
```

Comment:
This shouldn't be as highlighted as it is.  It's an "emergency measure" only.  It's not good behaviour.

Reply:
Possibly remove it entirely, as it's not about running a server - maybe under Client Hints and Tips, with a note that it's for times when there are no available servers with no one on, really.

Reply:
@gilgongo Did you add this to "move people away" from setting up private servers? I think this should still be documented somewhere, but maybe massively shortened.

Reply:
Move it to the FAQ?

Reply:
Or tips and tricks?

Reply:
On second thoughts, yes - T&T. The FAQ says is really for more general stuff.

Reply:
And link that from line 23.

Reply:
Ok. Please see my latest commits